recipes-extended/spi-minnowmax-board/files/modules-load.d/at86rf230.conf.sample

```diff
@@ -0,0 +1,12 @@
+# These modules are required to enable AT86RF230 802.15.4 radio in the
+# MinnowBoard MAX
+#
+# To enable loading, link as below:
+#
+# requirement: ln -s /lib/modprobe.d/spi-minnow-at86rf230.conf.sample /etc/modprobe.d/spi-minnow-at86rf230.conf
+# install: ln -s /usr/lib/modules-load.d/at86rf230.conf.sample /etc/modules-load.d/at86rf230.conf
+#
+# NOTE: MinnowBoard MAX BIOS must have LPSS SPI enabled.
+spi_pxa2xx_platform
+spi-minnow-at86rf230
+spi-minnow-board
```

Comment:
@yongli3 one more question. is the main driver loaded or should we also add at86rf230 .ko to this list?

Ditto for cc2520 below.